integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/datamap/TestDataMapCommand.scala

```diff
@@ -204,6 +204,37 @@ class TestDataMapCommand extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test("test show datamap: show datamap property related information") {
+    val tableName = "datamapshowtest"
+    val datamapName = "bloomdatamap"
+    val datamapName2 = "bloomdatamap2"
+    val datamapName3 = "bloomdatamap3"
+    sql(s"drop table if exists $tableName")
+    sql(s"create table $tableName (a string, b string, c string) stored by 'carbondata'")
```

Comment:
add a test case for preagg also, with DM properties and without DMProperties behavior

Reply:
It seems that preaggregate datamap does not support user specified DMProperties. Howwever it has DMProperties internally.
In the current commit, it will show: 
```
+-----------+------------+--------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
|DataMapName|ClassName   |Associated Table                |DataMap Properties                                                                                                                      |
+-----------+------------+--------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
|datamap1   |preaggregate|default.datamapshowtest_datamap1|'CHILD_SELECT QUERY'='c2VsZWN0IGNvdW50KGEpIGZyb20gZGF0YW1hcHNob3d0ZXN0', 'QUERYTYPE'='AGGREGATION', '_internal.deferred.rebuild'='false'|
+-----------+------------+--------------------------------+----------------------------------------------------------------------------------------------------------------------------------------+
```

Do you think it's OK?